lib/jsdom/living/aborting/AbortSignal-impl.js

```diff
@@ -26,7 +26,14 @@ class AbortSignalImpl extends EventTargetImpl {
       algorithm();
     }
     this.abortAlgorithms.clear();
-    this._dispatch(Event.createImpl(["abort"]), false);
+
+    this._dispatch(Event.createImpl(
+      [
+        "abort",
+        { bubbles: false, cancelable: false }
```

Comment:
We shouldn't need to explicitly set bubbles and cancelable, I think. But no big deal. I filed #2179 for future cleanup here.